docs/index.html

```diff
@@ -7,5 +7,5 @@
 	</head>
 	<body>
 		<div id="app"></div>
-	<script type="text/javascript" src="build/bundle.a53ec907.js"></script></body>
+	<script type="text/javascript" src="build/bundle.a22ef900.js"></script></body>
```

Comment:
So each time, when we add new component, we need to update the build version?

Reply:
this is documentation. since you add more document for added component, you need to build documentation to push them to github page.